Velaptor/OpenGL/ShaderCode/rectangle.frag

```diff
@@ -313,7 +362,7 @@ void main()
     float halfHeight = shape.w / 2.0;
     bool isFilled = pass_isFilled > 0.0;
 
-    // Clamp the corner radius
+    // Clamp the border thickness
```

Comment:
Do all of these type of comments need to be changed to describe what it does vs. what to do? 
```suggestion
    // Clamps the border thickness
```